cmd/release-tools/release.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+script_name="${0##*/}"
+script_dir="$(dirname $(realpath -s "$0"))"
+release_tool="${script_dir}/release-tool"
+owner=${OWNER:-clearcontainers}
```

Comment:
Might be worth making this something like the following to make it easier to switch to kata?

```
# github organisation that owns the repository
project=clearcontainers

owner=${OWNER:-$project}
```